tests/src/Context/MinkContext.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\joinup\Context;
+
+use Behat\Mink\Driver\GoutteDriver;
+use Drupal\DrupalExtension\Context\MinkContext as DrupalExtensionMinkContext;
+
+/**
+ * Provides step definitions for interacting with Mink.
+ */
+class MinkContext extends DrupalExtensionMinkContext {
+
+  /**
+   * Attach File to field.
+   *
+   * @param string $field
+   *   Field name.
+   * @param string $path
+   *   File path.
+   */
+  public function attachFileToField($field, $path) {
```

Comment:
Why is this code needed? Its purpose is undocumented. This is overriding `MinkContext::attachFileToField()` for some reason.